jwst/source_catalog/source_catalog.py

```diff
@@ -24,12 +25,12 @@
                                     SourceCatalog)
 from photutils.aperture import (CircularAperture, CircularAnnulus,
                                 aperture_photometry)
-from photutils.utils._wcs_helpers import _pixel_scale_angle_at_skycoord
 
 from jwst import __version__ as jwst_version
 
 from .. import datamodels
 from ..datamodels import ImageModel, ABVegaOffsetModel
+from ._wcs_helpers import pixel_scale_angle_at_skycoord
```

Comment:
Not a big fan of maintaining two versions of a function in two different places (in this case one in `photutils.utils` and now a second here in `source_catalog`). Is there no other way around this, other than making big changes to `photutils` itself?

Reply:
I agree in principle, but here I was importing a private function from `photutils`.  Because it's private, there's no guarantee that it will be stable (or even continue to exist).  In this case, the function API *did* change such that the pipeline would break for the next photutils release unless I copied the code here.  The solution is to make the `photutils` function public, but that couldn't happen until the next release.  Unfortunately, this is the best solution for now.

Reply:
Does this resolve https://github.com/spacetelescope/jwst/issues/6220 ?

Reply:
Cool. I added it to the list of things this PR fixes in the top comment.